### ISSUE TITLE
Lower Bound gets reset in Include Range filter

### ISSUE THREAD

**Issue by bhogan-mitre** [plotly/react-chart-editor#554]
One other weird observation is that pressing `<tab>` after changing the Upper Bound sometimes clears out the Lower Bound.

![2018-07-05_15-40-23](https://user-images.githubusercontent.com/32336739/42344337-09a365dc-806a-11e8-96cf-8fa644fe8d7d.png)

To reproduce: switch Operator to "Include Range", click to focus on Upper Bound and change value from default 0, then click (or `<tab>`) focus away from Upper Bound and Lower Bound gets reset.

**Comment by dmt0** [comment]
Couldn't reproduce :(

**Comment by bhogan-mitre** [comment]
Bummer, thanks for trying.  For what its worth, I'm using Chrome on a Mac, Version 67.0.3396.99 (Official Build) (64-bit).

**Comment by dmt0** [comment]
Yep, same here, Chrome 67 on Mac. Behaves as expected. You're saying it happens "sometimes". If you find a specific scenario that triggers the issue, we'd sure look at it. Thanks!

**Comment by bhogan-mitre** [comment]
Yes, I will let you know if I have more specific observations.  It's not a huge concern however.
